Sources/Mockolo/Executor.swift

```diff
@@ -208,6 +213,7 @@ class Executor {
                          to: outputFilePath,
                          loggingLevel: loggingLevel,
                          concurrencyLimit: concurrencyLimit,
+                         captureAllFuncArgsHistory: shouldCaptureAllFuncArgsHistory,
```

Comment:
Can we rename this to "enableArgsHistory"?  
-Capture sounds a bit like a capture list in a closure.
-Also no need to specify "Func" as args are passed into init only once, so it's already implied that it's a func.

And move this argument up after declType; we want to put bool params close to each other.

Reply:
> Can we rename this to "enableFuncArgsHistory" or "enableArgsHistory"? Capture sounds a bit like a capture list in a closure.
Sure. I will rename it to `enableArgsHistory`!

> And move this argument up after declType; we want to put bool params close to each other.
OK 😃

Reply:
I moved this after `useMockObservable` and renamed in d621414.